Makefile

```diff
@@ -222,6 +223,18 @@ tag: ## Tag topolvm images.
 		--tag $(IMAGE_PREFIX)topolvm-with-sidecar:$(IMAGE_TAG) \
 		$(IMAGE_PREFIX)topolvm-with-sidecar:$(ORIGINAL_IMAGE_TAG)
 
+##@ Chart Testing
+
+.PHONY: ct-lint
+ct-lint: ## Lint and validate a chart.
+	docker run \
+		--network host \
+		--workdir /data \
+		--volume ~/.kube/config:/root/.kube/config:ro \
+		--volume $(shell pwd):/data \
+		quay.io/helmpack/chart-testing:v$(CHART_TESTING_VERSION) \
+		ct lint --config ct.yaml
+
```

Comment:
I only succeeded to run `make ct-lint` without setting `--charts charts/topolvm`, or running `git config --global --add safe.directory /data` before running `ct lint`. Could you tell me how to run this make target without above additional options?

Reply:
I have an additional comment. `--network host` and `--volume ~/.kube/config:/root/.kube/config:ro` are unnecessary for `ct lint`. They are only necessary for `ct install`.

Reply:
> Could you tell me how to run this make target without above additional options?

I only run `make ct-lint` command like the below:

```
$ make ct-lint
docker run \
        --workdir /data \
        --volume /root/go/src/github.com/topolvm/topolvm:/data \
        quay.io/helmpack/chart-testing:v3.7.1 \
        ct lint --config ct.yaml
Linting charts...

------------------------------------------------------------------------------------------------------------------------
 Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 topolvm => (version: "11.0.1", path: "charts/topolvm")
------------------------------------------------------------------------------------------------------------------------

"jetstack" has been added to your repositories
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "jetstack" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading cert-manager from repo https://charts.jetstack.io
Deleting outdated charts
Linting chart "topolvm => (version: \"11.0.1\", path: \"charts/topolvm\")"
Validating /data/charts/topolvm/Chart.yaml...
Validation success! 👍
==> Linting charts/topolvm
[INFO] Chart.yaml: icon is recommended
[WARNING] templates/lvmd/psp.yaml: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
[WARNING] templates/node/psp.yaml: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
[WARNING] templates/scheduler/psp.yaml: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+

1 chart(s) linted, 0 chart(s) failed
------------------------------------------------------------------------------------------------------------------------
 ✔︎ topolvm => (version: "11.0.1", path: "charts/topolvm")
------------------------------------------------------------------------------------------------------------------------
All charts linted successfully
```

Reply:
As we talked about before, I removed the ct-lint command as it is not needed in CI.

Reply:
@satoru-takeuchi I've applied what you pointed out in your comment in the below link.
Is it possible for you to check again to see if this will stop the error from occurring?

https://github.com/topolvm/pvc-autoresizer/pull/169#discussion_r1090148402